hail/python/hail/vds/combiner/variant_dataset_combiner.py

```diff
@@ -701,6 +709,16 @@ def maybe_load_from_saved_path(save_path: str) -> Optional[VariantDatasetCombine
                     "entry types. Overwriting with types from supplied VDS.")
         gvcf_reference_entry_fields_to_keep = ref_entry_tmp
 
+        all_entry_types = chain(vds.reference_data._type.entry.items(),
+                                vds.variant_data._type.entry.items())
+        call_fields_tmp = {name for name, typ in all_entry_types if typ == hl.tcall} - {'LGT', 'GT'}
+        if 'LPGT' in call_fields_tmp:
+            call_fields_tmp = (call_fields_tmp - {'LPGT'}) | {'PGT'}
+        if set(call_fields) != call_fields_tmp:
+            warning("Mismatch between 'call_fields' and VDS call fields. "
+                    "Overwriting with types from supplied VDS")
```

Comment:
Include the mismatch. In particular, the user should know the `call_fields` argument but might not know what's in the dataset.


What do you mean by "overwriting with types ..."? It's not the types that are taken from the VDS but the call fields that are taken from it, right?

Reply:
Will do. Also will clarify.